### PR TITLE
[ICDS Dashboard] Changed the incremented aggregation level to original value for undernutrition data

### PR DESCRIPTION
**Pull request by challabeehyv**
https://dimagi-dev.atlassian.net/browse/ICDS-1129

##### SUMMARY
Changed the incremented aggregation level to original value for undernutrition data

##### FEATURE FLAG
No

### CODE REVIEW

File: custom/icds_reports/views.py [hunk 1]
```diff
@@ -533,6 +533,10 @@ def get(self, request, *args, **kwargs):
         config.update(get_location_filter(location, domain))
         loc_level = get_location_level(config.get('aggregation_level'))
 
+        # Changing the aggregation level to the original convention which is 1 for state,
```
Comment (Rohit25negi) [inline custom/icds_reports/views.py]: Can you make a #TODO comment here to make this change more reusable to other places this works well.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Small comment.

File: custom/icds_reports/views.py [hunk 2]
```diff
@@ -533,6 +533,10 @@ def get(self, request, *args, **kwargs):
...
+        #  2 for district, 3 for block, 4 for sector and 5 for awc to optimise the db query
+        if config['aggregation_level'] > 1:
```
Comment (Rohit25negi) [inline custom/icds_reports/views.py]: setting the aggregation_level this way is fine for this case as we testing the speed improvement. But we need to make sure we do it in a reusable way if this works well and do it other places as well

Comment (challabeehyv) [reply custom/icds_reports/views.py]: get_location_filter is aactually incrementing the aggregation value, we need to check and remove the addition if not necessary. Added a TODO

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**